aten/src/ATen/cuda/CUDABlas.cpp

```diff
@@ -745,8 +745,11 @@ void gemm_and_bias(
     epilogue = CUBLASLT_EPILOGUE_GELU_BIAS;
 #endif
   }
-  computeDesc.setAttribute(CUBLASLT_MATMUL_DESC_EPILOGUE, epilogue);
-  computeDesc.setAttribute(CUBLASLT_MATMUL_DESC_BIAS_POINTER, bias);
+
+  if (bias) {
```

Comment:
I see, you're trying to use nullptr to indicate optionality.

Reply:
Just for clarity I'd prefer an explicit comparison with nullptr and a comment.

Having said that, we usually use a different pattern (the optional utility) to indicate optionality.

I'm not sure if we want to insist on that here. cc @albanD @malfet

Reply:
In these very low level kernels where everything is raw pointers, we don't usually use Optional anymore.

Reply:
@cpuhrsch  @albanD  Let me check against nullptr to make it more readable.